src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -214,13 +228,14 @@ def __from_api_server_node(
             address for address in addresses if "internalip" in address.type.lower()
         ]
         internal_ip = ",".join([addr.address for addr in internal_addresses])
+        node_taints = api_server_node.spec.taints or []
         taints = ",".join(
-            [cls.__to_taint_str(taint) for taint in api_server_node.spec.taints]
+            [cls.__to_taint_str(taint) for taint in node_taints]
```

Comment:
I see .__to_taint_str still get a Hikaru Taint. does it work?

Reply:
(same for __to_active_conditions_str)

Reply:
Probably a bug. Nice catch